### PR TITLE
GET feedback/review returning public reviews without authorization

### PR DESCRIPTION
**Pull request by tommygonzaleza**

### CODE REVIEW

File: breathecode/feedback/views.py [hunk 1]
```diff
@@ -326,23 +326,22 @@ def get_review_platform(request, platform_slug=None):
...
 
 
-class ReviewView(APIView):
+@api_view(['GET'])
+@permission_classes([AllowAny])
+def get_reviews(request):
     """
     List all snippets, or create a new snippet.
     """
+    items = Review.objects.filter(is_public=True)
```
Comment (jefer94) [inline breathecode/feedback/views.py]: filter by status done

Comment (jefer94) [reply breathecode/feedback/views.py]: `total_rating is None or total_rating >= 0 and <= 10`

Comment (jefer94) [reply breathecode/feedback/views.py]: `total_rating__isnull=False or comments__isnull=False`

File: breathecode/feedback/views.py [hunk 2]
```diff
@@ -326,23 +326,22 @@ def get_review_platform(request, platform_slug=None):
...
+    lookup = {}
```
Comment (jefer94) [inline breathecode/feedback/views.py:336]: https://breathecode-test.herokuapp.com/admin/feedback/review/784/change/?_changelist_filters=status__exact%3DDONE this commentary is positive, but the grade ain't positive, it not have sense, investigate why

File: breathecode/feedback/views.py [hunk 3]
```diff
@@ -326,23 +326,22 @@ def get_review_platform(request, platform_slug=None):
...
 
-    def get(self, request, format=None):
-
-        items = Review.objects.filter(is_public=True)
-        lookup = {}
-
-        items = items.filter(**lookup).order_by('-created_at')
+    items = items.filter(**lookup).order_by('-created_at')
```
Comment (jefer94) [inline breathecode/feedback/views.py:338]: Investigate the range of `total_rating` if each platform use a custom range, the information must be sanity before can be used

**Review by jefer94 — CHANGES_REQUESTED** [review]

**Review by alesanchezr — APPROVED** [review]

**Pull request merged**